data_warehouse/services/training_hours_statistics_service.py

```diff
@@ -0,0 +1,72 @@
+'''培训学时与工作量统计模块'''
+from django.db.models import (Count, Sum, F)
+from data_warehouse.services.coverage_statistics_service import (
+    CoverageStatisticsService
+)
+from auth.services import UserService
+
+
+class TrainingHoursStatisticsService:
+    '''培训学时与工作量统计服务'''
+    @staticmethod
+    def get_training_hours_data(user, start_time, end_time):
+        '''返回院系的培训学时数据
+        Parameters
+        ------
+        user: User
+            当前登录的用户，必须为管理员。
+            若为校级管理员，则导出所有T3院系的数据，若为院系管理员，则导出该部门的数据。
+        start_time: datetime
+            统计范围的开始时间
+        end_time: datetime
+            统计范围的结束时间
+
+        Returns
+        ------
+        list of dict
+            {
+                key1: department,
+                    部门名称
+                key2: total_users,
+                    学院总人数
+                key3: total_coveraged_users,
+                    在指定时间范围内存在培训记录的总人数
+                key3: total_hours,
+                    总培训学时
+            }
+        '''
+        department_id = None
+        if user.is_department_admin:
+            department_id = user.administrative_department.id
+        records = CoverageStatisticsService.get_training_records(
```

Comment:
You should prepare `start_time` and `end_time` in your serializers, and `CoverageStatisticsService.get_training_records()` should require these data, that's what serializers do.